cmd/get/schema/schema.go

```diff
@@ -0,0 +1,111 @@
+package schema
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/loginradius/lr-cli/api"
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/loginradius/lr-cli/config"
+	"github.com/loginradius/lr-cli/request"
+
+	"github.com/spf13/cobra"
+)
+
+var all string
+
+type Schema struct {
+	Display          string `json:"Display"`
+	Enabled          bool   `json:"Enabled"`
+	IsMandatory      bool   `json:"IsMandatory"`
+	Parent           string `json:"Parent"`
+	ParentDataSource string `json:"ParentDataSource"`
+	Permission       string `json:"Permission"`
+	Name             string `json:"name"`
+	Rules            string `json:"rules"`
+	Status           string `json:"status"`
+	Type             string `json:"type"`
+}
+type schemaStr struct {
+	Data []Schema `json:"Data"`
+}
+
+var url string
+
+func NewschemaCmd() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     "schema",
+		Short:   "get schema config",
+		Long:    `This commmand lists schema config`,
+		Example: heredoc.Doc(`$ lr get schema`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if all == "" {
+				return &cmdutil.FlagError{Err: errors.New("`opt` is required argument")}
+			}
+			return get()
+
+		},
+	}
+	fl := cmd.Flags()
+	fl.StringVarP(&all, "opt", "o", "", "options to get active fields or all fields")
+	return cmd
+}
+
+func get() error {
+	res, err1 := api.GetSites()
+	if res.Userlimit == 7000 {
```

Comment:
What other things we are getting to check plan details?

Reply:
We can also check the plan details with `Domainlimit` attribute from `SitesReponse` struct.